modules/useQueries.js

```diff
@@ -1,15 +1,22 @@
-import qs from 'qs'
+import warning from 'warning'
+import { parse, stringify } from 'query-string'
 import runTransitionHook from './runTransitionHook'
 import parsePath from './parsePath'
 
 const SEARCH_BASE_KEY = '$searchBase'
 
 function defaultStringifyQuery(query) {
-  return qs.stringify(query, { arrayFormat: 'brackets' }).replace(/%20/g, '+')
+  return stringify(query).replace(/%20/g, '+')
 }
 
-function defaultParseQueryString(queryString) {
-  return qs.parse(queryString.replace(/\+/g, '%20'))
+const defaultParseQueryString = parse
+
+function isNestedObject(object) {
+  for (var p in object)
```

Comment:
Nits:
- This should probably be `const`
- `hasOwnProperty` guard? (ESLint can actually be set to warn against that)